### DISCUSSION TITLE
Design of JSME

### DISCUSSION THREAD

**Discussion by Vision20202** [jsme-editor/jsme-editor.github.io#4] [Ideas]
First of all I would like to say big thanks to Peter Ertl and Bruno Bienfait for JSME. It is really the best molecular editor!
I have a suggestion/question. Is it possible to change design of JSME like in attached picture?

![JSME](https://user-images.githubusercontent.com/72662623/111295112-11e0f200-865c-11eb-816e-c1456098b63c.png)

**Comment by jsme-editor** [comment]
Here is the closer to your suggestion that I could get:
![image](https://user-images.githubusercontent.com/75544418/111866127-87560680-896b-11eb-9e44-76744c3411f7.png)


The initialization code:
```javascript
       function jsmeOnLoad() {
            jsmeEditor = new JSApplet.JSME("jsme_container", "600px", "400px",
                {"options": "newlook",
                    "guicolor": "#FFFFFF",
                });

        }

```

**Comment by Vision20202** [comment]
Thank you. Is it possible to change color of each elements? I prefer black color for all of them.

**Reply by jsme-editor** [reply]
The next version will have this capability:
![image](https://user-images.githubusercontent.com/75544418/111869153-356aac00-897e-11eb-8b1b-8b20279040eb.png)


The initialization code:

```javascript
        function jsmeOnLoad() {
            jsmeEditor = new JSApplet.JSME("jsme_container", "600px", "400px",
                {"options": "newlook",
                    "guicolor": "#FFFFFF",
                    "guiAtomColor": "#000000"
                });

        }
```

**Comment by angel-herraez** [comment]
Hello. Thanks again for this piece of cake.
Is there a possibility to configure the left-side set of buttons so they display some other element types? Even replacing some of the existing ones.
I know I can go for "X" but I need a quick access to other frequent elements in a particular design.
Thanks
